.goreleaser.brew.yml

```diff
@@ -18,3 +18,6 @@ brew:
   dependencies:
     - kubernetes-cli
     - aws-iam-authenticator
+
+  custom_block: |
+    head https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_Darwin_amd64.tar.gz
```

Comment:
you might need to add double quote here as well :sob:

Reply:
Thanks Tam. I thought I did but I forgot to push the changes.